src/main/kotlin/no/nav/klage/kaka/clients/norg2/Norg2Client.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.klage.kaka.clients.norg2
+
+import brave.Tracer
+import no.nav.klage.kaka.exceptions.EnhetNotFoundForSaksbehandlerException
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.bodyToMono
+
+
+@Component
+class Norg2Client(
+    private val norg2WebClient: WebClient,
+    private val tracer: Tracer
+) {
+
+    @Value("\${spring.application.name}")
+    lateinit var applicationName: String
+
+    fun hentEnhet(enhetsnummer: String): Norg2Enhet? {
```

Comment:
Tror vi burde gå for engelsk `getEnhet` her (og generelt). Alternativt bli enige om noe annet.

Reply:
Enig i det, retter det.